containerd/main.go

```diff
@@ -137,6 +142,15 @@ func main() {
 				}
 			}
 		}
+		if logLevel := context.GlobalString("log-level"); logLevel != "" {
+			lvl, err := logrus.ParseLevel(logLevel)
+			if err != nil {
```

Comment:
You can use

```
if err != nil {
    lvl = logrus.InfoLevel
    ...
}

logrus.SetLevel(lvl)
```

Also, could you update the error message to say that the level is being defaulted to info?

Thanks